docs/manuals/studio/gui.md

```diff
@@ -72,6 +72,147 @@ set to the property is proposed to the user:
   <img src="../images/autocomplete_function.gif" width="75%"/>
 </p>
 
+## Page preview
+
+If you are used to working with Markdown in Visual Studio Code, you certainly came across the
+[Markdown preview](https://code.visualstudio.com/docs/languages/markdown#_markdown-preview)
+feature: pressing `Ctrl+Shift+V` in an editor that contains Markdown content creates a window
+showing the representation of a Markdown file. If the text editor and preview window are displayed
+next to each other, you are able to instantly see the changes done in the active Markdown file.
+
+This feature is leveraged by Taipy Studio so that Taipy visual elements are recognized and rendered
+in the Markdown preview window. When a valid Taipy GUI visual element is inserted in the Markdown
+file, it appears in the Preview section, and the properties that were set are reflected in a way
+that is very similar to what you will achieve in the application itself.
+
+Here is an animation showing how this works.
+
+<p align="center">
+  <img src="../images/preview_start.gif" width="100%"/>
+</p>
+
+A new Markdown file is created, then some basic Markdown content is added. The preview area
+displays the result of interpreting the Markdown text.<br/>
+If you add a Taipy visual element to the Markdown file, Taipy Studio assumes that this file
+can be used as a page definition, and renders the Taipy element in the preview area.<br/>
+Note that elements appear to be active (in the example above, the button can be pressed) but
+that is only a visual effect. No action is really triggered.
+
+### Non literal properties
+
+Some Taipy GUI visual elements have properties that accept properties that do not have a literal
+type, or can be interpreted.
+
+<h5>List of values</h5>
+
+Elements such as [`selector`](../gui/viselements/selector.md) or
+[`toggle`](../gui/viselements/toggle.md) have a property called `lov` that can be set to a
+series of strings. This would be properly handled by the Taipy Studio preview:
+
+<p align="center">
+  <img src="../images/preview_lov.png" width="80%"/>
+</p>
+
+<h5>Images</h5>
+
+You can specify the image displayed by a [`image`](../gui/viselements/image.md) control in the
+preview area by providing the path to the image file, relative to the root directory of the
+project:
+
+<p align="center">
+  <img src="../images/preview_image.png" width="80%"/>
+</p>
+
+### Mock values
+
+The Markdown preview pane of Taipy Studio supports the binding of a property to a value.<br/>
+
+Because the Visual Studio code preview area is not connected to an actual application that could
+provide values set to variables, Taipy Studio uses a TODO taipy.mock.json
+
+<p align="center">
+  <img src="../images/preview_data.png" width="80%"/>
+</p>
+
+!!! warning "No expression evaluation"
+    Expressions are not evaluated: you can use "{value}" to evaluate the variable in the preview
+    area, but expressions (such as "{value+1}") will **not** be replaced with their evaluation.
```

Comment:
You can set value+1 in mock file --> it can be replaced but not evaluated

Reply:
Oh. I did not know that.

Reply:
Actually @dinhlongviolin1 this does not seem to work.
I tried:

`file.md`:
```
<|{x}|slider|>

Double: <|{x*2}|input|>
```
with

`taipy.mock.json`:
```
{
  "x": 12,
  "x*2": 24
}
```

and the preview still shows an input control with "{x*2}" in it...